com.mudeprolinux.korerorero.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.mudeprolinux.korerorero
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: korerorero
+
+finish-args:
+  # X11 and IPC access
+  - --socket=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # PulseAudio
+  - --socket=pulseaudio
+
+modules:
+  - name: Kōrerorero
+    buildsystem: simple
+    build-commands:
+      - install -D korerorero.py /app/bin/korerorero
+      - install -Dm644 korerorero.png /app/share/icons/hicolor/512x512/apps/com.mudeprolinux.korerorero.png
+      - install -Dm644 korerorero.desktop /app/share/applications/com.mudeprolinux.korerorero.desktop
+      - install -Dm644 com.mudeprolinux.korerorero.appstream.xml /app/share/appdata/com.mudeprolinux.korerorero.appdata.xml
+    sources:
+      - type: git
+        path: https://github.com/tobagin/korerorero.git
```

Comment:
this is not how you specify a remote git repository. also a tag or commit is required.